spec/support/multiline_literal_brace_layout_trailing_comma_examples.rb

```diff
@@ -3,20 +3,12 @@
 RSpec.shared_examples_for 'multiline literal brace layout trailing comma' do
   let(:prefix) { '' } # A prefix before the opening brace.
   let(:suffix) { '' } # A suffix for the line after the closing brace.
-  let(:open) { nil } # The opening brace.
-  let(:close) { nil } # The closing brace.
   let(:a) { 'a' } # The first element.
   let(:b) { 'b' } # The second element.
 
   # same line message for symmetrical style (use [...] to abbreviate).
-  let(:same_line_message) do
-    'Closing brace must be on the same line as the last element when opening [...]'
-  end
 
   # same line message for same_line style
-  let(:always_same_line_message) do
-    'Closing brace must be on the same line as the last element.'
-  end
```

Comment:
I understand these and [these](https://github.com/rubocop/rubocop/pull/11877/files#diff-36013affefc5b0625e2e90546b0252028199ee2462616abd7858b4c025d32705L8-L9) might be intended to be left for documentation and I can leave them if preferred.